src/Components/Facility/Consultations/Feed.tsx

```diff
@@ -294,11 +307,13 @@ export const Feed: React.FC<IFeedProps> = ({ consultationId }) => {
     fullScreen: () => {
       if (!(screenfull.isEnabled && liveFeedPlayerRef.current)) return;
       !screenfull.isFullscreen
-        ? screenfull.request(
-            videoWrapper.current
-              ? videoWrapper.current
-              : liveFeedPlayerRef.current
-          )
+        ? isIOS
+          ? screenfull.request(
+              videoWrapper.current
+                ? videoWrapper.current
+                : (liveFeedPlayerRef.current as HTMLElement)
+            )
+          : screenfull.request(findDOMNode(liveFeedPlayerRef.current) as any)
```

Comment:
why are we doing it differently for iOS and other devices, is this condition necessary?

and also shouldn't it be the other way around?
```suggestion
        ? !isIOS
          ? screenfull.request(
              videoWrapper.current
                ? videoWrapper.current
                : (liveFeedPlayerRef.current as HTMLElement)
            )
          : screenfull.request(findDOMNode(liveFeedPlayerRef.current) as any)
```